code/game/objects/structures/curtain.dm

```diff
@@ -6,9 +6,10 @@
 	var/icon_type = "bathroom"//used in making the icon state
 	color = "#ACD1E9" //Default color, didn't bother hardcoding other colors, mappers can and should easily change it.
 	alpha = 200 //Mappers can also just set this to 255 if they want curtains that can't be seen through
-	layer = SIGN_LAYER
+	plane = GAME_PLANE_UPPER
+	layer = ABOVE_MOB_LAYER
```

Comment:
```suggestion
```
Planes override layers for things not on the same plane, so setting layer is not required